packages/builder/builder-rspack-provider/src/core/createCompiler.ts

```diff
@@ -26,10 +26,11 @@ export async function createCompiler({
       timings: true,
     });
 
-    obj.children?.forEach(c => {
-      c.time &&
-        logger.success(`${c.name} compiled successfully in`, c.time, 'ms');
-    });
+    !stats.hasErrors() &&
```

Comment:
It is recommended that `&&` is only used in short statements, we should use `if (!stats.hasErrors())` here.